### PR TITLE
Sn 541

### PR DESCRIPTION
**Pull request by siddirp**

### CODE REVIEW

File: python/pylib/data_sets.py [hunk 1]
```diff
@@ -90,7 +90,7 @@
 DID_DUAL_IMU_RAW_MAG            = 84
 DID_DUAL_IMU_MAG                = 85
 DID_PREINTEGRATED_IMU_MAG       = 86
-DID_WHEEL_CONFIG                = 87
+DID_GROUND_VEHICLE                = 87
```
Comment (aabogdanov) [inline python/pylib/data_sets.py:93]: Is wheel_config now a part of ground_vehicle?

File: src/data_sets.h [hunk 2]
```diff
@@ -1985,12 +2008,24 @@ typedef struct PACKED
 
 } wheel_config_t;
```
Comment (aabogdanov) [inline src/data_sets.h:2009]: If ground_vehicle_t replaced wheel_config_t, do we still need wheel_config_t? Does it have any use/purpose anywhere else in the code?

**Review by waltjohnson — APPROVED** [review]

File: python/pylib/data_sets.py (hunk 1 above)
Comment (waltjohnson) [reply python/pylib/data_sets.py:93]: Yes.

File: src/data_sets.h (hunk 2 above)
Comment (waltjohnson) [reply src/data_sets.h:2010]: wheel_config_t is used in the nvm_flash_config_t and also inside of ground_vehicle_t.  So we need both.

**Pull request merged**